ansible/README.md

```diff
@@ -15,36 +14,86 @@ You will need your public key deployed to the host for ssh access
 
 You may need to connect to the host once with ssh to make sure it's in your list of known hosts
 
+If you install [casey/just](https://github.com/casey/just), then some tasks
+will be simplified.
+
 ## Before Running
 
-Create secret files under secrets-[env]:
+**Setup the Ansible Vault Passphrase**
+There are passwords, secret keys, and other sensitive information required for everything
+to work. These secrets are included in the repo and they are encrypted using
+ansible-vault.  To run ansible-successfully, you will need the passphrase to
+decrypt them.  Ask a team member. Then, run:
```

Comment:
Can there be only one password?

Reply:
All files encypted by ansible-vault use the same password.

Reply:
guess that's one advantage of git-crypt but it was also a pain in the ass and this is still better than passing around files in slack so :+1:  seems like a good move in the right direction